terraform/jenkins-master/configure-master/.env

```diff
@@ -0,0 +1,11 @@
+export CLOUD_NAME="OE-ACC"
```

Comment:
With terraform, we launch from an independent box, I am thinking this configuration can be changed to sourcing local environment variables to allow this to be configurable?

There is also the question of if we should be exposing information like our subscription information to the public.